### ISSUE TITLE
Fails to run with solidity 0.6.0

### ISSUE THREAD

**Issue by tsudmi** [cgewecke/eth-gas-reporter#191]
Hi,

It looks like the library does not work with solidity 0.6.0:
```
Error: invalid fragment object (argument="value", value={"stateMutability":"payable","type":"receive"}, version=abi/5.0.0-beta.142)
    at Logger.makeError (/home/user/dapp/node_modules/@ethersproject/logger/lib/index.js:105:21)
    at Logger.throwError (/home/user/dapp/node_modules/@ethersproject/logger/lib/index.js:114:20)
    at Logger.throwArgumentError (/home/user/dapp/node_modules/@ethersproject/logger/lib/index.js:117:21)
    at Function.Fragment.fromObject (/home/user/dapp/node_modules/@ethersproject/abi/lib/fragments.js:369:23)
    at Function.Fragment.from (/home/user/dapp/node_modules/@ethersproject/abi/lib/fragments.js:350:25)
    at /home/user/dapp/node_modules/@ethersproject/abi/lib/interface.js:73:41
    at Array.map (<anonymous>)
    at new Interface (/home/user/dapp/node_modules/@ethersproject/abi/lib/interface.js:72:60)
    at utils.getContractNames.filter.forEach.name (/home/user/dapp/node_modules/eth-gas-reporter/lib/gasData.js:72:27)
    at Array.forEach (<anonymous>)
    at files.forEach.file (/home/user/dapp/node_modules/eth-gas-reporter/lib/gasData.js:45:10)
    at Array.forEach (<anonymous>)
    at GasData.initialize (/home/user/dapp/node_modules/eth-gas-reporter/lib/gasData.js:41:11)
    at Runner.Gas.runner.on (/home/user/dapp/node_modules/eth-gas-reporter/index.js:45:16)
    at Runner.emit (events.js:203:15)
    at start (/home/user/dapp/node_modules/mocha/lib/runner.js:848:10)
    at Runner.run (/home/user/dapp/node_modules/mocha/lib/runner.js:876:5)
    at Mocha.run (/home/user/dapp/node_modules/mocha/lib/mocha.js:612:17)
    at resolve (/home/user/dapp/node_modules/truffle/build/webpack:/packages/core/lib/test.js:139:1)
    at new Promise (<anonymous>)
    at Object.run (/home/user/dapp/node_modules/truffle/build/webpack:/packages/core/lib/test.js:138:1)
```
eth-gas-reporter: 0.2.14
truffle: 5.1.12

**Comment by cgewecke** [comment]
@tsudmi Yes, it relies on a js parser dependency which is in the process of being updated for the newer syntax. Should be fixed this week. 

Thanks for opening.

**Comment by cgewecke** [comment]
@tsudmi This should be (mostly) fixed with `0.2.15`.  Have migrated the parser to [consensys/solidity-parser-diligence][1]. 

The only 0.6.x syntax which I've seen not working there is casting address to payable like this:
```solidity
payable(someAddress)
```

[1]: https://github.com/consensys/solidity-parser-antlr

**Comment by tsudmi** [comment]
Hopefully, there will be support for that as well in the future.
Thanks @cgewecke
